src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -65,11 +79,21 @@ public async Task WarmupInvoke(HttpContext httpContext)
 
             ReadRuntimeAssemblyFiles();
 
+            WorkerWarmup();
+
             await WarmUp(httpContext.Request);
```

Comment:
Wondering if we should rename this to `HostWarmup` so we're more specific with the introduction of the new worker warmup method?